docs/theme/static/js/components/upcoming-event/upcoming-event.js

```diff
@@ -0,0 +1,71 @@
+/* global moment */
+import $ from 'jquery';
+// import moment from 'moment'; // added globally
```

Comment:
Not a big deal, but you could still import moment even though you're using a script tag by setting it as `instantiated`. See http://stealjs.com/docs/StealJS.loading-from-cdn.html#always-loading-from-a-cdn if you want to do that.